vscode/src/circuit.ts

```diff
@@ -337,8 +337,8 @@ export function updateCircuitPanel(
     ? `${params.operation.operation} with ${params.operation.totalNumQubits} input qubits`
     : projectName;
 
-  // Trim the Q#: prefix from the target profile name - that's meant for the ui text in the status bar
-  const target = `Target profile: ${getTargetFriendlyName(targetProfile).replace("Q#: ", "")} `;
+  // Trim the QIR: prefix from the target profile name - that's meant for the ui text in the status bar
+  const target = `Target profile: ${getTargetFriendlyName(targetProfile).replace("QIR: ", "")} `;
```

Comment:
Since you're essentially removing the extra prefix, I think we can just remove the `.replace` call completely